equinix/README.md

```diff
@@ -47,7 +47,13 @@ The `harvester.install.config_url=https://metadata.platformequinix.com/userdata`
       password: p@ssword  # replace with a your password
     install:
       mode: create
-      mgmt_interface: eth0
+      networks:
+        harvester-mgmt: # The management bond name. This is mandatory.
+          bond_option:
+            mode: balance-tlb
```

Comment:
Is `mode: balance-tlb` mandatory? If not, will `balance-tlb` be the default mode if absent?

Reply:
Default to `balance-tlb` if missing. I am fine whether put it here or remove it.

Reply:
We can leave it here. I was just curious. Though generally speaking, configs that has default values does not need to be illustrated in the examples unless they are relevant to the situation.